### PR TITLE
DTSPO-16593 - updated AKS upgrade doc

### PR DESCRIPTION
**Pull request by cpareek**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/DTSPO-16593

### Change description ###
updated AKS upgrade documetation

### CODE REVIEW

File: source/aks/upgrading-aks-clusters.html.md.erb [hunk 1]
```diff
@@ -86,7 +91,7 @@ kubectl get pdb -A
 - To list all failed helm releases
 
 ```command
-kubectl get hr --all-namespaces | grep False
+kubectl get hr --all-namespaces | awk '/(Unknown|False)/'
```
Comment (thomast1906) [inline source/aks/upgrading-aks-clusters.html.md.erb:94]: Do we change this to != True?

Comment (cpareek) [reply source/aks/upgrading-aks-clusters.html.md.erb:95]: make sense

Comment (thomast1906) [reply source/aks/upgrading-aks-clusters.html.md.erb]: Something like that?
`kubectl get hr | awk '$3 != "True"' `

**Review by thomast1906 — CHANGES_REQUESTED** [review]

File: source/aks/upgrading-aks-clusters.html.md.erb (hunk 1 above)
Comment (cpareek) [reply source/aks/upgrading-aks-clusters.html.md.erb]: think should be like this
`kubectl get hr --all-namespaces | awk '!/(True)/'`

**Review by thomast1906 — APPROVED** [review]

**Pull request merged**